parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# Update Labels for Kubernetes nodes
+
+set -euo pipefail
+
+# TODO(charliedmcb): replace template <Node Name>. Still trying to find how to access the current node's name. 
+kubectl label --overwrite nodes <Node Name> $KUBELET_NODE_LABELS
```

Comment:
seems that the node name is just $(hostname)

Reply:
I wasn't able to confirm if hostname is actually correct, but that makes sense and I think you are right. Switching it back now. 

Funny enough if it is right my original template envvar was correct, as $NODE_NAME is set to $(hostname) in cse_config.sh